src/main/ruby/truffleruby/core/truffle/concurrent_hash_map.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+module TruffleRuby
+  class ConcurrentHashMap
+    def initialize(options = nil)
```

Comment:
`options` isn't used.

Reply:
The `Map` child class which will inherit this class [uses the options params](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent-ruby/concurrent/map.rb#L125) so not having it will raise ArgumentError (which happened when I was testing it)